scripts/metadata_checks.py

```diff
@@ -36,7 +36,7 @@
                 "type": "comparison",
                 "left": {"type": "field", "path": "metadata.resource_type.id"},
                 "operator": "==",
-                "right": "publication-thesis",
```

Comment:
can you note it down for deployment to change the value in the DB (for the existing check already)? Do you have deployment recipe?